docs/content/mongocxx-v3/working-with-bson.md

```diff
@@ -32,10 +32,23 @@ The various methods of creating BSON documents and arrays are all
 equivalent. All interfaces will provide the same results, the choice of
 which to use is entirely aesthetic.
 
+## List builder {#list}
+
+The simplest way to create a BSON document or array is to use the JSON-like
+list builder: 
+
+```c++
+// { "hello": "world" }
+bsoncxx::builder::list list_builder = {"hello", "world"};
+bsoncxx::document::view document = list_builder.view().get_document();
```

Comment:
There's currently no way to extract a document. I created [CXX-2131](https://jira.mongodb.org/projects/CXX/issues/CXX-2131?filter=allopenissues) to address this.